tests/cross/src/test/scala/tests/hover/HoverErrorSuite.scala

```diff
@@ -13,4 +13,21 @@ class HoverErrorSuite extends BaseHoverSuite {
        |""".stripMargin,
     ""
   )
+
+  check(
+    "error",
+    """|final case class Dependency(
+       |    org: String,
+       |    name: Option[String],
+       |    version: Option[String]
+       |)
+       |
+       |object Dependency {
+       |  def <<ap@@ply>>(org: String) = Dependency(org, None, None)
+       |  def apply(org: String, name: String) = Dependency(org, Some(name), None)
```

Comment:
I could have simplified this further, but just lazy copy pasted when I created the issue. The second apply can just be removed in all of these as well.

Reply:
Removed!